_posts/2017-03-01-pulp-at-scale-2017.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Pulp at SCALE '17
+author: Brian Bouterse
+---
+Pulp is participating in the Southern CAlifornia Linux Expo
+([SCALE](http://www.socallinuxexpo.org/scale/15x)) from March 2-5, 2017 in Pasadena, California.
+Michael Hrivnak will be giving a talk titled *[Deploying a Multi-Service App in Kubernetes](
+http://www.socallinuxexpo.org/scale/15x/presentations/deploying-multi-service-app-kubernetes)* which
+deploys Pulp itself using containers and Kubernetes. Brian Bouterse will be giving a talk titled
```

Comment:
It seems weird to talk about yourself in the third person in an authored message.